extensions/ql-vscode/src/remote-queries/remote-queries-interface.ts

```diff
@@ -16,10 +15,12 @@ import {
 import { Logger } from '../logging';
 import { getHtmlForWebview } from '../interface-utils';
 import { assertNever } from '../pure/helpers-pure';
-import { commandRunner } from '../commandRunner';
+import { AnalysisResult, RemoteQueryResult } from './remote-query-result';
+import { RemoteQuery } from './remote-query';
+import { RemoteQueryResult as RemoteQueryResultViewModel } from './view/remote-query-result';
+import { AnalysisResult as AnalysisResultViewModel } from './view/remote-query-result';
```

Comment:
It would be nice to be more explicit about where the shared code is between the extension and the webview. 

Elsewhere, this shared code goes into the `src/pure` directory. Can you move these files there?

I would like to create a codeql query that asserts that the webviews only import modules that are either `pure` or in a `view` directory.

Reply:
Would it be possible to apply the rule in all view directories (including subdirs), rather than just the top-level one? Then these files could stay where they are. If not then I'll just move them - but I do like the fact that they're close to all the other remote query stuff.

Reply:
It's probably better to keep these files co-located with the rest of remote-queries. How about making a new directory called `shared` or something and putting them there?